### PR TITLE
Add subtitles and icons in app permissions

### PR DESCRIPTION
**Pull request by vdnj**
Capture d’écran 2022-11-24 à 09 37 22

### CODE REVIEW

File: src/locales/fr.json [hunk 1]
```diff
@@ -463,7 +463,9 @@
     "monthly_statistics": "Statistiques mensuelles",
     "monthly_stats_phrase": "Compte rendu de la sortie de données depuis votre cozy vers %{app} le %{date}:",
     "exit_rights": "Droits de sortie",
-    "uninstall": "Désinstaller"
+    "uninstall": "Désinstaller",
+    "exit_rights_secondary": "La donnée peut sortir hors du Cozy, elle ne sera alors plus maintenue secrète",
+    "limited_right_access_secondary": "La donnée est accessible en aveugle, elle est maintenue secrète dans votre Cozy"
```
Comment (paultranvan) [inline src/locales/fr.json:468]: Je trouve la formulation un peu alambiquée. Que pensez-vous de "La donnée est uniquement accessible localement, elle ne sort jamais de votre Cozy." @poupotte @vdnj ?

**Review by paultranvan — APPROVED** [review]
The code looks code despite 2 nits. But I have some concerns about the wording

File: src/components/Permissions/AccessRightsSection.jsx [hunk 2]
```diff
@@ -30,11 +33,27 @@ const AccessRightsSection = ({
...
+                ? t('Permissions.limited_right_access')
+                : t('Permissions.exit_rights')}
+            </Typography>
+          }
+          secondary={
+            !isRemoteDoctypes
+              ? t('Permissions.limited_right_access_secondary')
+              : t('Permissions.exit_rights_secondary')
+          }
+          primaryTypographyProps={{ color: 'red' }}
```
Comment (paultranvan) [inline src/components/Permissions/AccessRightsSection.jsx:55]: Why is this necessary? I don't see any red in the screenshot (and I'm not sure we should use hardcoded colors)

Comment (vdnj) [reply src/components/Permissions/AccessRightsSection.jsx:55]: my bad, this was for testing and was not working. I remove it

**Review by paultranvan — APPROVED** [review]
